providers/src/airflow/providers/ydb/CHANGELOG.rst

```diff
@@ -31,6 +31,24 @@ Changelog
   This release removes YDBScanQueryOperator from this provider package.
   At this point, YDBExecuteQueryOperator could load unlimited amount of rows, so no specific operator is needed.
 
+1.5.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Migrate YDB Operator to new DBAPI (#43784)``
```

Comment:
I think that was **somewhat** breaking despite the comment https://github.com/apache/airflow/pull/43784/files#r1838110622 - maybe worth increasing it to 2.0.0 anyway?

Reply:
We can. I relayed on Oleg judgment as the mantainer of the upstream dependency